extensions/data-workspace/src/common/workspaceTreeDataProvider.ts

```diff
@@ -25,6 +25,10 @@ export class WorkspaceTreeDataProvider implements vscode.TreeDataProvider<Worksp
 		this._onDidChangeTreeData?.fire();
 	}
 
+	close(): void {
+		vscode.commands.executeCommand('workbench.action.closeFolder');
```

Comment:
I suppose there is only one command for closing workspace and folder, right?

Reply:
correct - they're the same command